src/components/common/ModalPicker.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from "react";
+import { StyleSheet, View, Picker, TouchableOpacity, Text } from "react-native";
+import { CRButton } from "src/components/common";
+
+class ModalPicker extends Component {
```

Comment:
favor functional components over classes

Reply:
Completed. Couldn't figure it out how to do it without state because of the RN Picker.

Reply:
if you need state then use a class, if you don't need it just use a function

Reply:
I needed a class, so a class it is.